### DISCUSSION TITLE
Menus Configuration

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#429] [General]
# Menus Configuration

This article describes the properties of the menu entry and how to config the menus.

https://hbstack.dev/en/docs/configuration/menus/

**Comment by bigkevuk** [comment]
How can I get the top menu to be across the site like your screenshots vs under the ... button top right? Think I may have missed something here.

**Reply by razonyang** [reply]
The header menus is supported by the header module, see https://hbstack.dev/en/modules/header/menus/ for details.
